Fixing typo (#2445)
opensearch-project/project-website: _posts/2023-11-24-sessions-at-reinvent.md

---
layout: post
title:  "OpenSearch Sessions at re:Invent"
authors:
  - dtaivpp
date: 2023-11-24
categories:
 - news
meta_keywords: re:Invent, OpenSearch Sessions, open source, Amazon OpenSearch Service
meta_description: Sessions you will want to attend at re:Invent 2023 to learn more about OpenSearch!
excerpt: Wondering what sessions to attend at re:Invent to learn more about OpenSearch? We've got you covered! Here are the top 6 sessions to attend this year at re:Invent to deepen your OpenSearch knowledge.
---

<a href="https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=OpenSearch">![OpenSearch Sessions at re:Invent Banner](/assets/media/blog-images/2023-11-24-sessions-at-reinvent/reInvent_banner.jpg){: .img-fluid }</a>

> With re:Invent right around the corner I wanted to share a few of the OpenSearch sessions that are going to be particularly interesting. These are broken up a bit between open-source OpenSearch and the Amazon OpenSearch Service but both should be applicable no matter how you run OpenSearch.

Reviewer: ```suggestion
With re:Invent right around the corner, I want to share a few of the OpenSearch sessions that are going to be particularly interesting. Some sessions pretain to open-source OpenSearch or the Amazon OpenSearch Service, but both should be applicable no matter how you run OpenSearch.
```

# 1. OPN311 - Extending OpenSearch

Time: Nov. 27, 10:00 AM - 11:00 AM (PST)
<br>Location: [Wynn, Level 1, Palmer 2](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=OPN311)

> Stephen Crawford and Daniel Doubrovkine (or DB) will be diving into how you can extend OpenSearch. If you are looking to create an extension, register a step for a search or ingestion pipeline, or building your own plugin you will know where to start after this chalk talk!

Reviewer: ```suggestion
Stephen Crawford and Daniel Doubrovkine (or DB) dive into how you can extend OpenSearch. If you are looking to create an extension, register a step for a search or ingestion pipeline, or are building your own plugin, this chalk talk will help you get started!
```

# 2. ANT312 - Using Amazon OpenSearch Service as a vector database for gen AI apps

Time: Nov. 30, 11:30 AM - 1:30 PM (PST)
<br>Location: [Caesars Forum, Level 1, Forum 118](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT312)

> All anyone wants to discuss anymore is Generative AI. Well we have you covered here as well! In this session Arun Lakshmanan and Jianwei Li will walk you through how to use OpenSearch as a context store for your Generative AI applications! They will walk you through what tools OpenSearch provides to build a comprehensive solution with Generative AI.

Reviewer: ```suggestion
If all you want to talk about is Generative AI, this session has you covered.  Arun Lakshmanan and Jianwei Li walk you through how to use OpenSearch as a context store for your Generative AI applications and what tools OpenSearch provides to build a comprehensive solution with Generative AI.
```

# 3. OPN202 - Logging with Amazon EKS, Bottlerocket, Fluent Bit, and OpenSearch

Time: Nov. 29, 3:00 PM - 5:00 PM (PST)
<br>Location: [Venetian, Level 3, Murano 3203](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=OPN202)

> I am excited to be running this workshop with Kyle Davis, Senior Developer Advocate for Bottlerocket. We will cover how you can do observability on a shell-less cluster. You will deploy an EKS cluster onto nodes running Bottlerocket, a security hardened container focused operating system. Then we will deploy OpenSearch and Fluent Bit to do observability on our cluster. This is an end to end workshop with no prior experience required so we hope to see you there!

Reviewer: ```suggestion
I am excited to be running this end-to-end workshop with Kyle Davis, Senior Developer Advocate for Bottlerocket. In this workshop, we'll cover how you can perform observability on a shell-less cluster. To start, we'll deploy an EKS cluster onto nodes running Bottlerocket, a security hardened container focused operating system. Then, we'll deploy OpenSearch and Fluent Bit to do observability on our cluster. No prior experience is required so we hope to see you there!
```

# 4. ANT210 - Improve your search with vector capabilities in OpenSearch Service

Time: Nov. 30, 2:30 PM - 3:30 PM (PST)
<br>Location: [Caesars Forum, Level 1, Summit 201](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT210)

> Did you know that OpenSearch is not just a vector database but 3 vector databases in one? With OpenSearch you can leverage vector stores like FAISS, nmslib, and Lucenes Vector Store. In this session Achal Kumar from Intuit along with Aruna Govindaraju and Jon Handler from AWS will talk through how to get more relevant search results with OpenSearch as a vector store. Additionally, you will get to hear first hand how Intuit is using Amazon OpenSearch Service to get more relevant results.

Reviewer: ```suggestion
Did you know that OpenSearch is not just a vector database but three vector databases in one? With OpenSearch, you can leverage vector stores like FAISS, nmslib, and Lucene's Vector Store. In this session, Achal Kumar from Intuit along with Aruna Govindaraju and Jon Handler from AWS talk through how to get more relevant search results with OpenSearch as a vector store. Additionally, you'll hear first hand how Intuit is using Amazon OpenSearch Service to get more relevant results.
```

# 5. COM310 - Processing over 450 million events per day in real time with AWS

Time: Nov. 27, 5:30 PM - 5:50 PM (PST)
<br>Location: [Venetian, Level 2, Hall B, Expo, Booth #1480, Theater 3](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=COM310)

> As an observability solution, OpenSearch scales! Hear from Neylson Crepalde at A3Data how they are doing observability on a whopping 5Tb per month of logs. He will cover the architecture used and many of their learnings along the way.

Reviewer: ```suggestion
As an observability solution, OpenSearch scales! Hear from Neylson Crepalde at A3Data about how they are doing observability on a whopping 5Tb per month of logs. He'll cover the architecture he used and the many lesson they learned along the way.
```

# 6. ANT301 - What’s new in Amazon OpenSearch Service

Time: Nov. 30, 4:00 PM - 5:00 PM (PST)
<br>Locations:
* [Caesars Forum, Level 1, Summit 201](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT301)
* [Mandalay Bay, Level 2, South, Mandalay Bay Ballroom L, Content Hub, Teal Screen](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT301-SC1)
* [MGM Grand, Level 1, Boulevard 163, Content Hub, Yellow Screen](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT301-SC2)

> And saving the best for last; make sure you checkout live what is new in the Amazon OpenSearch Service! This session hosted by Mukul (GM of Search Services at AWS), Carl (Director of OpenSearch at AWS), and Bill Burket (Engineering Leader at EA), covers many of the new things that are going to make your experience building on OpenSearch even better! You are not going to want to miss this session.

Reviewer: ```suggestion
And saving the best for last; make sure you checkout live what is new in the Amazon OpenSearch Service! This session hosted by Mukul (GM of Search Services at AWS), Carl (Director of OpenSearch at AWS), and Bill Burket (Engineering Leader at EA), covers many of the new things that are going to make your experience building on OpenSearch even better! You are not going to want to miss this session.
```

Author: @Naarcha-AWS hey, this change looks the same as the original text.
